libs/styles/src/lib/scss/components/_datepicker.scss

```diff
@@ -0,0 +1,141 @@
+$datepicker-border-color: #9799A4;
+$datepicker-border-radius: 4px;
+$datepicker-font-size: 16px;
+$datepicker-width: 125px;
+$datepicker-height: 40px;
+$datepicker-color: #34394C;
+
+@mixin cvi-datepicker($base: &) {
+  --comment: 'A datepicker';
+
+  @at-root {
+    #{$base}__input {
+      &-container {
+        border: 1px solid $datepicker-border-color;
+        border-radius: $datepicker-border-radius;
+        display: flex;
+        align-items: center;
+        width: $datepicker-width;
+        height: $datepicker-height;
+
+        &-focused {
+          border-color: get-color(sapphire-blue-10);
+        }
+      }
+
+      &-field {
+        font-size: $datepicker-font-size;
+        border-left: 1px;
+        border-radius: $datepicker-border-radius;
+        padding: 8px;
+        width: 96px;
+
+        &:focus {
+          outline: none;
+        }
+      }
+
+      &-icon-wrapper {
+        width: 29px;
+        padding-top: 6px;
+      }
+
+      #{$base}--is-disabled & {
+        background-color: get-color(black-coral-0);
+        border-color: get-color(black-coral-5);
+      }
+    }
+
+    #{$base}__calendar {
+      box-sizing: border-box;
+      position: absolute;
+      background: get-color(white);
+      border: 1.25px solid get-color(black-coral-6);
+      border-radius: $datepicker-border-radius;
+      width: 392px;
+      height: 280px;
+      color: $datepicker-color;
+
+      &-weekend {
+        color: grey !important;
+      }
+
+      &-visually-hidden {
```

Comment:
The library has got a mixin, CSS class, and a component all of which can be used to achieve the same effect, might make more sense to use those since the implementation might be more robust across browsers. The mixin is `cvi-screenreader-text`, class is `cvi-screenreader-text`, and component is `ScreenreaderTextComponent`.